src/pages/api/sepana/delete.page.ts

```diff
@@ -0,0 +1,45 @@
+import { NextApiHandler } from 'next'
+
+import { deleteSepanaDoc, sepanaAlert } from './utils'
+
+const handler: NextApiHandler = async (req, res) => {
+  const { id } = req.query
+
+  if (typeof id !== 'string' || !id.length) {
+    res.status(400).end()
+    return
+  }
+
+  try {
+    await deleteSepanaDoc(id).then(async _res => {
+      await sepanaAlert({
+        type: 'alert',
+        alert: 'DELETED_RECORD',
+        body: `ID: ${id}`,
+      })
+
+      res.status(200).json(_res.data)
+    })
+  } catch (error) {
+    await sepanaAlert({
+      type: 'alert',
+      alert: 'DB_UPDATE_ERROR',
+      body: JSON.stringify(error),
+    })
+
+    res.status(500).json({
```

Comment:
**suggestion** i think this needs a return

Reply:
doesn't look like handlers need to return a value https://nextjs.org/docs/api-routes/response-helpers

Reply:
@peripheralist even if we `catch` here, the following `sepanaAlert` on line 37 will still execute right? Is this expected?

Reply:
even though we've given a response, i believe the function will continue executing as normal.

Reply:
oh i see what you mean now. actually, there shouldn't be any clause after the try/catch, was an error in copying over some code from DeleteAll. fixing